src/main/java/ch/njol/skript/ScriptLoader.java

```diff
@@ -759,6 +796,12 @@ public String run(final Matcher m) {
 					deleteCurrentEvent();
 				}
 				
+				// Remove the script from the disabled scripts list
+				@SuppressWarnings("null")
+				File disabledFile = new File(file.getParentFile(), "-" + file.getName());
+				if (disabledFiles.contains(disabledFile))
+					disabledFiles.remove(disabledFile);

```

Comment:
you can actually just do `disabledFiles.remove(disabledFile)` without checking if it's inside the set.
From the docs: `Removes the specified element from this set if it is present (optional operation)`